### ISSUE TITLE
Confirm Barcode Format

### ISSUE THREAD

**Issue by dianamartin** [cityofaustin/atd-data-tech#574]
_As a warehouse person, I need the ability to scan a barcode that looks for that inventory item and finds it in the system_

**Comment by dianamartin** [comment]
Found this in Knack help guides: [Use a Scanner to Search and Update](https://support.knack.com/hc/en-us/articles/360015666351-Can-I-use-a-barcode-or-credit-card-scanner-with-Knack-)

**Comment by johnclary** [comment]
@dianamartin to confirm that warehouse staff are using stock number barcodes

**Comment by dianamartin** [comment]
@johnclary Tested today at 8900

Hardware used: 
- Dell Laptop
- USB Adesso barcode scanner

Plugged USB scanner into laptop and tested filter ability on the"
- `Inventory` page
- `Cycle Count` page

The scanner was having some issues with scanning the barcode. It's in a plastic label holder with a magnet on the back. I think the plastic is slightly obscuring the barcode. We tried slipping it out of the label holder and it scanned the paper perfectly. Ray and I talked and said maybe we need to change out the labels to a laminated thing. 

Won't move on anything until we finalize the barcode scanner hardware.
